src/main/java/listeners/OnlinerTestListener.java

```diff
@@ -0,0 +1,124 @@
+package listeners;
+
+import io.qameta.allure.Attachment;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.testng.ITestContext;
+import org.testng.ITestListener;
+import org.testng.ITestResult;
+import webdriver.DriverManager;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class OnlinerTestListener implements ITestListener {
+    public static final String LOGFILE_NAME = "logfile.log";
+    private static final Logger logger = Logger.getLogger(OnlinerTestListener.class);
+    private final VideoRecorder recorder = new VideoRecorder();
+
+    @Attachment(value = "logfile", type = "text/plain")
+    public static byte[] addFileToAllureReport() throws IOException {
+        return Files.readAllBytes(Paths.get(LOGFILE_NAME));
+    }
+
+    @Attachment(value = "video", type = "video/mp4")
+    public static byte[] addVideoToAllureReport(File file) throws IOException {
+        return Files.readAllBytes(Paths.get(String.valueOf(file)));
+    }
+
+    public static void clearTheFile(File file) throws IOException {
```

Comment:
Investigate different utils, i am sure a method with similar functionality is already implemented somewhere.